### ISSUE TITLE
I can set the Version Tag property of a Decision

### ISSUE THREAD

**Issue by romansmirnov** [camunda/camunda-modeler#802]
It is possible to specify a `camunda:versionTag` on the `<decision... />`:

```xml
<decision id="versionTag" name="VersionTag DMN" camunda:versionTag="1.0.0">
  <decisionTable id="decisionTable">
    ...
  </decisionTable>
</decision>
```
---

Also see [SUPPORT-4084](https://app.camunda.com/jira/browse/SUPPORT-4084)

**Comment by philippfromme** [comment]
Implemented by https://github.com/bpmn-io/dmn-js-properties-panel/commit/73d3124183dcd8ee0d6dca8ee52ccbf10e0e828a.

**Comment by nikku** [comment]
Closed via 267c44a67f1d97fe5fc74e6ed7b05e67642b5321.
